docs/configuration.rst

```diff
@@ -179,6 +180,10 @@ directory into the container at the same path.
 If a referenced environment variable is not set, Scuba exits with a
 configuration error.
 
+Volume host paths can be absolute or relative. If a relative path is used, it
+is interpreted as relative to the directory in which ``.scuba.yml`` is found.
+To avoid ambiguity, relative paths must start with ``./`` or ``../``.
+
```

Comment:
In [line 335 of config.py](https://github.com/JonathonReinhart/scuba/pull/227/files#diff-1bd8088b035e8e07af4ead68414dce30741e7cedbdef91de1a3d70e048ae1992R335), it specifies that container paths must be absolute. That seems like a useful note to put in this user-facing documentation as well.